src/imports.rs

```diff
@@ -608,7 +608,7 @@ impl UseTree {
                             span: self.span,
                             list_item: None,
                             visibility: self.visibility.clone(),
-                            attrs: None,
+                            attrs: self.attrs.clone(),
```

Comment:
I can't come up with a specific use case at the moment, but have some concerns about this having unintended knock on effects (even if not now, perhaps later on).

Would you mind updating this just a bit so that this is only done conditionally with the `Item` granularity variant? Should be feasible to do so by updating the function signature to take a param (boolean, ImportGranularity, etc.) or attaching the granularity variant to the `UseTree`, etc. and feel free to leave an inline comment on this line (or wherever the attrs are conditionally derived) explaining that we only want to do this for the Item variant